### ISSUE TITLE
Buy cable glands

### ISSUE THREAD

**Issue by nathanielrindlaub** [tnc-ca-geo/sci-cameratrap-project#4]
I believe we'll need 3 glands to seal wires coming into the pelican case: 

1. 1x single-slotted for Ethernet cable (confirm we'll be using Cat6)
2. 1x double-slotted for the MC4 cables (one negative wire and and one positive)
3. 1x single-slotted for the antenna cable

**Comment by nathanielrindlaub** [comment]
@postfalk can you do a sanity check on the list above for me before I figure out the wire dimensions and order them? My questions are:

- Am I missing anything or are there other wires that need to leave the Pelican case
- Are we using cat6 cable to connect the base station to the network and also do we have cat6 on the island? I haven't bought any cable or any RJ45 plugs...

**Comment by postfalk** [comment]
We are using cat 6. Will check the other stuff tomorrow. But:

a) Gary will be with us since Palmyra is now very distant and he can just go deep into his supplies.

b) It is always good to have some wiggle room and options on those things.

________________________________

**Comment by edisiENt** [comment]
Fewer holes in the Pelican case is better.  I suggest one for the power cables and Ethernet and a second for the antenna cable.

**Comment by nathanielrindlaub** [comment]
@edisiENt fewer the better makes sense to me, but for the initial temporary experiments we'll just be running the base off the UPS charge without solar, so my thinking was it would be nice to be able to seal up the power hole completely for that stage with one of these guys: https://www.automation24.com/sealing-insert-m25-pflitsch-uni-dicht-ufe-54pg

I could look for an antenna + Ethernet combo gland and a separate power gland though.

**Comment by nathanielrindlaub** [comment]
Done. I ordered:

- Ethernet - A M20 gland, locknut, and 7-9.5 mm single-hole insert for **CAT-6 (OD 7-9mm)**
- Antenna cable - A M25 gland, locknut, and a 9-11mm single-hole insert for **CA-400 (OD 10.29 mm)**
- Solar power cables - An M25 gland, locknut, and a handful of different inserts for **2x 12AWG (OD 4.7mm)**. 

Automation24 didn't have a cable glad sealing insert that would work for the pair of 12AWG MC4 cables. They don't have inserts with two small enough holes for the 12AWG (4.7 mm) in any of the M20, M25, or M32 plug sizes. I got a couple varieties I can mess around with to try to jerry-rig a good seal, as well as a some fully sealed inserts (no holes), that I can try to bore a custom hole through with one of these [tools](https://www.avogadro-lab-supply.com/item.php?item_id=455) used for boring holes in rubber laboratory stoppers. We'll see how I do...
